app/helpers/visit_helper.rb

```diff
@@ -0,0 +1,5 @@
+module VisitHelper
+  def when_to_expect_reply
+    # TODO
```

Comment:
I intended to port PrisonSchedule at the same time, but then realised we don't need it yet.  We might not need it at all.  I'll drop this until such time as we decide how we're going to handle this.